egui/src/input_state.rs

```diff
@@ -473,6 +473,8 @@ impl PointerState {
         let old_pos = self.latest_pos;
         self.interact_pos = self.latest_pos;
 
+        let mut delta_acc = Vec2::ZERO;
```

Comment:
this code can be simplified by removing `delta_acc` and just writintg to `self.delta` directly.

Reply:
Its my habit to leave variable local as long as I can, and do `global` mutations just after all actions. But yeah, I'll change that.